proto/common/common.proto

```diff
@@ -72,7 +76,7 @@ message ClientInfo {
   ClientType client_type = 1;
 
   // Used to indicate the type of traffic.  We can use this to prioritize resources.
-  // Next ID = 5.
+  // Next ID = 6.
```

Comment:
undo or set reserved...

Reply:
We should start catching more of these at the point they are introduced :-(